README.md

```diff
@@ -19,6 +19,14 @@ The reports section will not show up in the CMS if:
 
  * There are no reports to show
  * The logged in user does not have permission to view any reports
+ 
+For large datasets, the reports section may take a long time to load, since each report is getting a count of the items it contains to display next to the title.  
+You can mitigate this issue by setting a cap on the number of items that will be counted per report with the `limit_count_in_overview` configuration variable. You can either do for all reports, or if you know specifically which reports will be problematic you can set it for those specific reports.
```

Comment:
This is a bit ambiguous because of how github has chosen to show the excerpt, so to clarify I take it you mean the paragraph that starts "You can mitigate"?

Reply:
There was a line break before that paragraph using `  ` (two spaces) syntax - I'm changing this to a `\n` character for the sake of clarity.